service/lib/agama/software/manager.rb

```diff
@@ -132,11 +132,11 @@ def probe
         logger.info "Probing software"
 
         if repositories.empty?
-          start_progress(3)
+          start_progress_with_size(3)
```

Comment:
In this case the descriptions are known in advance. Should we use the `with_descritptions` constructor?